core/trino-spi/src/main/java/module-info.java

```diff
@@ -31,6 +31,7 @@
     exports io.trino.spi.memory;
     exports io.trino.spi.metrics;
     exports io.trino.spi.predicate;
+    exports io.trino.spi.protocol;
```

Comment:
Why does the SPI have a protocol package?

Reply:
This is for spooling manager SPI. We can move it to some other package if this is the wrong name for it.

Reply:
(SpoolingManager is pluggable)

Reply:
Yeah, let’s rename the package. That’s doesn’t seem like the right name.

Reply:
What do you propose?

Reply:
How about io.trino.spi.spool?